core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -3559,10 +3559,12 @@ protected function treeView()
 			}
 		}
 
+		$breadcrumb = (isset($GLOBALS['TL_DCA'][$table]['list']['sorting']['breadcrumb']) ? $GLOBALS['TL_DCA'][$table]['list']['sorting']['breadcrumb'] : '');
+
 		// Return if there are no records
 		if ($tree == '' && \Input::get('act') != 'paste')
 		{
-			return $return . '
+			return $return.$breadcrumb.'
```

Comment:
Maybe we should wrap the breadcrumb menu with `<div class="tl_formbody_edit"></div>`? Then we would not need extra CSS definitions.

Reply:
I was thinking that semantics do not fit when adding that div container.
But I like not changing the css also more.
Changed.